include/windows/config.h

```diff
@@ -0,0 +1,177 @@
+/* #undef HAVE_NETLINK_VERSION_H */
+
+/* psm provider is built */
+/* #undef HAVE_PSM */
+
+/* psm2 provider is built */
+/* #undef HAVE_PSM2 */
+
+/* psm2 provider is built as DSO */
+/* #undef HAVE_PSM2_DL */
+
+/* Define to 1 if you have the <psm2.h> header file. */
+/* #undef HAVE_PSM2_H */
+
+/* psm provider is built as DSO */
+/* #undef HAVE_PSM_DL */
+
+/* Define to 1 if you have the <psm.h> header file. */
+/* #undef HAVE_PSM_H */
+
+/* Define to 1 if you have the <rdma/rsocket.h> header file. */
+/* #undef HAVE_RDMA_RSOCKET_H */
+
+/* sockets provider is built */
+#define HAVE_SOCKETS 1
+
+/* sockets provider is built as DSO */
+/* #undef HAVE_SOCKETS_DL */
+
+/* Define to 1 if you have the <stdint.h> header file. */
+/* #undef HAVE_STDINT_H */
+
+/* Define to 1 if you have the <stdlib.h> header file. */
+#define HAVE_STDLIB_H 1
+
+/* Define to 1 if you have the <strings.h> header file. */
+#define HAVE_STRINGS_H 1
+
+/* Define to 1 if you have the <string.h> header file. */
+#define HAVE_STRING_H 1
+
+/* Define to 1 if compiler/linker support symbol versioning. */
+/* #undef HAVE_SYMVER_SUPPORT */
+
+/* Define to 1 if you have the <sys/mman.h> header file. */
+/* #undef HAVE_SYS_MMAN_H */
+
+/* Define to 1 if you have the <sys/stat.h> header file. */
+#define HAVE_SYS_STAT_H 1
+
+/* Define to 1 if you have the <sys/types.h> header file. */
+#define HAVE_SYS_TYPES_H 1
+
+/* Define to 1 if you have the <unistd.h> header file. */
+#define HAVE_UNISTD_H 1
+
+/* usnic provider is built */
+/* #undef HAVE_USNIC */
+
+/* usnic provider is built as DSO */
+/* #undef HAVE_USNIC_DL */
+
+/* verbs provider is built */
+/* #undef HAVE_VERBS */
+
+/* verbs provider is built as DSO */
+/* #undef HAVE_VERBS_DL */
+
+/* Define to 1 to enable valgrind annotations */
+/* #undef INCLUDE_VALGRIND */
+
+/* Define to the sub-directory where libtool stores uninstalled libraries. */
+#define LT_OBJDIR ".libs/"
+
+/* Name of package */
+#define PACKAGE "libfabric"
+
+/* Define to the address where bug reports for this package should be sent. */
+#define PACKAGE_BUGREPORT "ofiwg@lists.openfabrics.org"
+
+/* Define to the full name of this package. */
+#define PACKAGE_NAME "libfabric"
+
+/* Define to the full name and version of this package. */
+#define PACKAGE_STRING "libfabric 1.2.0"
+
+/* Define to the one symbol short name of this package. */
+#define PACKAGE_TARNAME "libfabric"
+
+/* Define to the home page for this package. */
+#define PACKAGE_URL ""
+
+/* Define to the version of this package. */
+#define PACKAGE_VERSION "1.2.0"
+
+/* Define to 1 if pthread_spin_init is available. */
+/* #undef PT_LOCK_SPIN */
+
+/* The size of `void *', as computed by sizeof. */
+#ifdef _WIN64
+#define SIZEOF_VOID_P 8
+#else /* _WIN64 */
+#define SIZEOF_VOID_P 4
+#endif /* _WIN64 */
+
+/* Define to 1 if you have the ANSI C header files. */
+#define STDC_HEADERS 1
+
+/* Version number of package */
+#define VERSION "1.2.0"
```

Comment:
Can you form this string using the FI_MAJOR/MINOR_VERSION numbers instead, so that it gets updated automatically?